rq/cli/cli.py

```diff
@@ -166,6 +167,9 @@ def worker(url, config, burst, name, worker_class, job_class, queue_class, path,
     cleanup_ghosts(conn)
     worker_class = import_attribute(worker_class)
     queue_class = import_attribute(queue_class)
+    exception_handlers = []
```

Comment:
`exception_handlers` should default to `None` instead of empty list. An empty list would disable the default behavior of moving failed jobs to `FailedQueue`.